addons/l10n_rs_edi/tests/test_xml_ubl_rs.py

```diff
@@ -0,0 +1,87 @@
+from odoo import Command
+from odoo.addons.l10n_account_edi_ubl_cii_tests.tests.common import TestUBLCommon
+from odoo.tests import tagged
+from odoo.tools import misc
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestUBLRS(TestUBLCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref="rs"):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.company_data['company'].write({
+            'country_id': cls.env.ref('base.rs').id,
+            'name': 'RS Company',
+            'city': 'Niš',
+            'zip': '12000',
+            'vat': 'RS101134702',
+            'phone': '+381 23 456 78 91',
+            'street': 'Nikole Pašića 30a',
+            'invoice_is_ubl_cii': True,
+        })
+        cls.company_data['company'].partner_id.l10n_rs_edi_registration_number = '87654321'
+
+        cls.currency_data['currency'] = cls.env.ref('base.RSD')
```

Comment:
Not needed since the `chart_template_ref` is already the correct one

Reply:
For some reason, the `self.currency_data['currency']` is not updated when the new `chart_template_ref` is set